api/ProtectedAudience.json

```diff
@@ -0,0 +1,72 @@
+{
+  "api": {
+    "ProtectedAudience": {
+      "__compat": {
+        "spec_url": "https://wicg.github.io/turtledove/#protectedaudience",
+        "support": {
+          "chrome": {
+            "version_added": "127"
+          },
+          "chrome_android": "mirror",
+          "edge": "mirror",
+          "firefox": {
+            "version_added": false
+          },
+          "firefox_android": "mirror",
+          "ie": {
+            "version_added": false
+          },
+          "oculus": "mirror",
+          "opera": "mirror",
+          "opera_android": "mirror",
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": "mirror",
+          "samsunginternet_android": "mirror",
+          "webview_android": "mirror",
+          "webview_ios": "mirror"
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "queryFeatureSupport": {
+        "__compat": {
+          "spec_url": "https://wicg.github.io/turtledove/#dom-protectedaudience-queryfeaturesupport",
+          "support": {
+            "chrome": {
+              "version_added": "127"
```

Comment:
Partial support seems to have landed in 122: https://github.com/chromium/chromium/commit/8f7613429532241b68b96671c56a7a0115459329

Support was extended in 126: https://github.com/chromium/chromium/commit/c9d50975b33200fb9fd8e93291c6c1180de76244#diff-28fd91f778c224914aede4eee2a04b9469d9b21e086b6eccd52bf417ea6bd275R23316